### ISSUE TITLE
Loss raising on validation set

### ISSUE THREAD

**Issue by semchan** [frostinassiky/gtad#22]
Dear, I try to retrain your code on the dataset you provide, but found that the loss always raising on validation dataset, could you please tell me why? thanks a lot

174it [04:30,  1.55s/it]
[Epoch 000]	Loss 6.12 = 4.85 + 1.27 (train)
204it [00:47,  4.27it/s]
[Epoch 000]	Loss 5.48 = 4.18 + 1.30 (validation)
current learning rate: 4e-05
174it [04:31,  1.56s/it]
[Epoch 001]	Loss 4.65 = 3.47 + 1.18 (train)
204it [00:47,  4.28it/s]
[Epoch 001]	Loss 5.53 = 4.27 + 1.26 (validation)
current learning rate: 4e-05
174it [04:32,  1.57s/it]
[Epoch 002]	Loss 4.01 = 2.89 + 1.12 (train)
204it [00:48,  4.23it/s]
[Epoch 002]	Loss 5.02 = 3.80 + 1.22 (validation)
current learning rate: 4e-05
174it [04:31,  1.56s/it]
[Epoch 003]	Loss 3.58 = 2.50 + 1.08 (train)
204it [00:47,  4.31it/s]
[Epoch 003]	Loss 7.76 = 6.55 + 1.21 (validation)
current learning rate: 4e-05
174it [04:37,  1.59s/it]
[Epoch 004]	Loss 3.49 = 2.44 + 1.05 (train)
204it [00:47,  4.33it/s]
[Epoch 004]	Loss 6.62 = 5.42 + 1.20 (validation)
current learning rate: 4.000000000000001e-07

**Comment by frostinassiky** [comment]
Hi @semchan Thanks for pointing out this!

I also met this question in my experiment. It is a common/open problem in temporal action detection.

1. THUMOS-14 dataset is not big enough. Thus, the model converges fast even if we set a small learning rate. If you did the same experiment on ActivityNet, the loss will go down for more epochs.
 
2. We cannot always expect a better detection result (in terms of mAP, AmAP) from a lower loss value because the metric and loss are not strongly correlated. Afaik, couples of work (e.g. DE:TR) on detection to tackle this issue by reporting the mAP of each epoch, which is more reasonable.

**Comment by semchan** [comment]
understood,thanks very much

---Original---
